src/arch/ArchHooks/ArchHooks_Unix.cpp

```diff
@@ -159,7 +161,8 @@ std::int64_t ArchHooks::GetMicrosecondsSinceStart( bool bAccurate )
 	timespec ts;
 	clock_gettime( g_Clock, &ts );
 
-	std::int64_t iRet = std::int64_t(ts.tv_sec) * 1000000 + std::int64_t(ts.tv_nsec)/1000;
+	//std::int64_t iRet = std::int64_t(ts.tv_sec) * 1000000 + std::int64_t(ts.tv_nsec)/1000;
```

Comment:
this is back to front, comment out the if statement, or you will force a slow update, its the if statement :)